pennylane/_qubit_device.py

```diff
@@ -1273,6 +1286,65 @@ def classical_shadow(self, wires, circuit, seed=None):
 
         return self._cast(self._stack([outcomes, recipes]), dtype=np.int8)
 
+    def shadow_expval(self, obs, circuit):
+        r"""Compute expectation values using classical shadows in a differentiable manner.
+
```

Comment:
This is a great docstring @Qottmann, although I do have a suggestion.

Since this is not user facing (instead, users should be redirected to `classical_shadow`), and due to the fact that the example here does actually use `QubitDevice.shadow_expval` directly, I recommend removing the example in this docstring, and instead directing users to `qml.shadow_expval` for more details.

This is for several reasons:

- It helps with maintainability; developers who update the `shadow_expval` docstring/usage/example likely won't remember to come here and update this one as well

- Due to the details in this docstring, it might start to rank highly in google searches, which we probably don't want.

Reply:
An example of this approach is NumPy's `np.ndarray.sum` docstring:

https://numpy.org/doc/stable/reference/generated/numpy.ndarray.sum.html

Noticed how brief it is, instead directly readers to the more user-facing `np.sum`.

Following this approach, we could even reduce the amount of theory and description here, and instead direct to `qml.shadow_expval` for usage and theory --- keeping here only the information regarding usage for device developers.

Reply:
💯 thanks for pointing that out, I confused the measurement and device method!